src/PostgREST/QueryBuilder.hs

```diff
@@ -418,7 +418,7 @@ pgFmtFilter table (Filter fld (Operation hasNot_ ex)) = notOp <> " " <> case ex
    (op, VText val) -> pgFmtFieldOp op <> " " <> case op of
      "like"  -> unknownLiteral (T.map star val)
      "ilike" -> unknownLiteral (T.map star val)
-     "@@"    -> "to_tsquery(" <> unknownLiteral val <> ") "
```

Comment:
If you run:
```sql
select to_tsvector('impossible') @@ 'impossible';
```
It'll output `false`, but if you run:
```sql
select to_tsvector('impossible') @@ to_tsquery('impossible');
```
It'll output a `true`, that's because:
```sql
SELECT 'impossible'::tsquery, to_tsquery('impossible');
   tsquery    | to_tsquery
--------------+------------
 'impossible' | 'imposs'
```
The `to_tsquery` is needed to build the lexemes, I think that `@@` just operates against strings and converts them to `tsquery` and for the case of `asdf, lala == to_tsquery(asdf), to_tsquery(lala)` the lexemes are the same.

So the `ts_query` is required the issue is that the tests are not catching that('foo/bar/qux' same as lexemes) so maybe you could improve the tests to catch that difference.